fluent-bit.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fluent-bit
+  version: 2.0.8
+  epoch: 0
+  description: Fast and Lightweight Log processor and forwarder
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache-2.0
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - flex
+      - glibc
+      - openssl-dev
+      - yaml-dev
+      - zlib-dev
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/fluent/fluent-bit/archive/v${{package.version}}.tar.gz
```

Comment:
We also have the source detained with checksums here:
- https://releases.fluentbit.io/2.0/source-2.0.8.tar.gz
- https://releases.fluentbit.io/2.0/source-2.0.8.tar.gz.sha256

(Other repo maintainers preferred that over Github tagged versions so just wanted to indicate it).